client/components/planning-editor-standalone/field-definitions/subject.ts

```diff
@@ -36,12 +36,12 @@ export function getSubjectField(): IFieldDefinition {
                 return (item.subject ?? []).map(({qcode}) => qcode);
             },
             storeValue: (item, operationalValue: Array<ISubjectCode['qcode']>) => {
-                const allSubjects: Array<ISubjectCode> = (planningApi.redux.store.getState().subjects ?? [])
+                const filteredSubjects: Array<ISubjectCode> = (planningApi.redux.store.getState().subjects ?? [])
```

Comment:
`filteredSubjects` isn't much better. It doesn't make sense to mention programming operations (filter,map,reduce etc.) that you performed - but what is being achieved by those programming operations. When it's something small and obvious like here I'd call it `subjects` simply without elaborating. If you wanted more details, then maybe `subjectsFull` since the purpose of the operation is to return full objects instead of qcodes only.